scene/main/node.cpp

```diff
@@ -217,6 +226,12 @@ void Node::_notification(int p_notification) {
 				memdelete(child);
 			}
 		} break;
+
+		case NOTIFICATION_TRANSLATION_CHANGED: {
+			if (is_inside_tree()) {
```

Comment:
```suggestion
			if (data.inside_tree) {
```
Mostly for consistency (though it's slightly more performant if the compiler does not inline it automatically).